postgres/step2.md

```diff
@@ -6,24 +6,34 @@ Deploy spire-server and spire-agent:
 
 # Wait for SPIRE to start
 
-Wait for both spire-agent and spire-server to be in Running state before
-continuing:
+Use the following command to check that both spire-agent and spire-server are in a Running state:
 
 `kubectl get pods -n spire`{{execute}}
 
-**Keep running the above `kubectl get` command until both pods are in the `Running`
-state.**
+Keep running the above `kubectl get` command until both pods are displayed as `Running`
+underneath `STATUS`.
+
+**It will take about 30 seconds for the SPIRE pods to start.**
 
 # SPIRE logs
 
-To look at spire-server logs:
+As you go through this demo, you may want to check the SPIRE Server
+and Agent logs to see how various actions affect each of them.
+
+To look at spire-server log:
 
 `kubectl logs -n spire --selector app=spire-server`{{execute}}
 
-To look at spire-agent logs:
+In the SPIRE Server log you should see a line similar to the following
+to indicate that SAT attestation has completed:
+`time="2020-05-30T00:13:12Z" level=info msg="Node attestation request completed" address="10.32.0.1:43428" attestor=k8s_sat method=node_api spiffe_id="spiffe://example.k8s.local/spire/agent/k8s_sat/example.k8s.local/642ae7cd-77af-4c95-99bd-ea815dd998a2" subsystem_name=node_api`
+
+To look at spire-agent log:
 
 `kubectl logs -n spire --selector app=spire-agent`{{execute}}
 
-**It will take about 30 seconds for SPIRE pods to start.**
+In the SPIRE Agent log you should see a line similar to the following
+to indicate that SAT attestation has completed:
```

Comment:
This log signifies initiation of SAT attestation not necessarily  the completion of it.